homeassistant/components/mqtt/discovery.py

```diff
@@ -154,18 +154,11 @@ def get_origin_support_url(discovery_payload: MQTTDiscoveryPayload) -> str | Non
 
 @callback
 def async_log_discovery_origin_info(
-    message: str, discovery_payload: MQTTDiscoveryPayload, level: int = logging.INFO
+    message: str, discovery_payload: MQTTDiscoveryPayload
 ) -> None:
     """Log information about the discovery and origin."""
-    # We only log origin info once per device discovery
-    if not _LOGGER.isEnabledFor(level):
-        # bail out early if logging is disabled
-        return
```

Comment:
We should probably leave this in so it doesn't have to construct the arguments if its not going to log

`get_origin_log_string(discovery_payload, include_url=True)`

https://docs.python.org/3/howto/logging.html#optimization

Reply:
Re-added